tools/elf2uf2/main.cpp

```diff
@@ -9,7 +9,12 @@
 #include <vector>
 #include <cstring>
 #include <cstdarg>
+#include <cstdint>
 #include <algorithm>
+#include "pico/platform.h"
+#define le_uint16_t stored_little_endian<uint16_t>
```

Comment:
this could be in platform.h

Reply:
Ok, I guess if you want an alternate implementation you could make your `#define`s before including `pico/platform.h` instead of after.